Doc/library/wave.rst

```diff
@@ -19,8 +19,8 @@ The :mod:`wave` module defines the following function and exception:
 
 .. function:: open(file, mode=None)
 
-   If *file* is a string, open the file by that name, otherwise treat it as a
-   file-like object.  *mode* can be:
+   If *file* is a string or :term:`path-like object`, open the file
+   by that name, otherwise treat it as a file-like object.  *mode* can be:
```

Comment:
Maybe add a reference to the "file-like object" term?

Reply:
I looked in the glossary and "file-like object" redirects to "file object" so I used that term instead.  Hope that that's okay.